codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt

```diff
@@ -99,9 +98,7 @@ class ServerProtocolTestGenerator(
         inputT to outputT
     }
 
```

Comment:
There's a few more calls to `RustWriter::write` instead of `RustWriter::rust` in this file (`codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt`). Do we want to change those?